### ISSUE TITLE
The DV media profile brand signaling convention is being specified by WAVE.

### ISSUE THREAD

**Issue by johnsim** [cta-wave/content-specification-task-force#19]
The Dolby Vision media profile submission is a reference to the ATSC published specification, but the small brand signaling convention is documented in WAVE. Will this impact the adoption of the media profile? Will it impact interop?

**Comment by rdoherty0** [comment]
Since the file brand is only useful to WAVE compliant clients, specifying this in the WAVE profile is appropriate and provides a completely referenceable specification for WAVE clients.  Hence, there should be no impact to adoption or interop since we are talking exclusively about WAVE clients.

**Comment by haudiobe** [comment]
WAVE does not define profiles - WAVE references profiles. We have not defined WAVE clients, so this is a new architectural concept.

**Comment by johnsim** [comment]
o Proposed resolution:   The CMAF binding spec must define the 4CC code for the Dolby Vision  Supplemental Data Profile (‘cdv1’) as well as document rules on how SEI  messages are included in CMAF representations.    **Before this publication takes place, we need to know the eventual  home for this CMAF binding information.    Once the eventual publication location is known, providing all other  issues are resolved, we should add the ‘cdv1’ binding information in the  Amendment 1 of our specification.  o Mr. Stockhammer noted he has a concern with this approach. It requires that  another organization includes whatever WAVE documents, and there is no  guarantee they will align with the WAVE spec. Mr. Poole shared this concern and  noted WAVE will need a strategy for dealing with that problem.**

**Comment by johnsim** [comment]
Agreed in October 24, 2018 conference call to postpone this issue until resolution of Supplemental Data Profile approval process - which is issue #23
